.github/release-check-action/RELEASE.md

```diff
@@ -0,0 +1,9 @@
+Release type: minor
+
+Implement parser handling incorrect input in DateTime scalar (#1035)
+
+Giving incorrect iso string to Time, Date or DateTime now raises a GraphQLError, without an original error.
+
+This is the same behaviour that graphql-core has: https://github.com/graphql-python/graphql-core/blob/b80085385dd4f818abb38c27e84f6fd16289684a/src/graphql/type/scalars.py#L37-L61
+
+The GraphQL error will include the error message raised by the string parser, eg `Value cannot represent a DateTime: "2021-13-01T09:00:00". month must be in 1..12`
```

Comment:
I have simplified the release notes a bit, but it is just a suggestion.

```suggestion
Matching the behaviour of `graphql-core`, passing an incorrect ISO string value for a Time, Date or DateTime scalar now raises a `GraphQLError` instead of the original parsing error.

The `GraphQLError` will include the error message raised by the string parser, e.g. `Value cannot represent a DateTime: "2021-13-01T09:00:00". month must be in 1..12`
```